en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc

```diff
@@ -0,0 +1,117 @@
+
+Provide the following connection information:
+
+include::_partials/connection-name.adoc[]
+
+Base URL::
+
+The default location URL for the cloud application that the connection uses.
+
+Authorization::
+
+The following authorization options are available:
+
+* *OAuth 2.0 Client Credentials* +
+  The OAuth 2.0 Client Credentials obtain an access token from a user-defined path. Provide a value for the following fields:
+  ** *Base URL*
+  ** *Client ID*
+  ** *Client Secret*
+  ** *Token URL*
+
+* *API Key* +
+  The API Key connects to applications without explicitly defining a user. This key must be added to all requests and the cloud application identifies the origin of the request and completes the authorization. Provide a value for the following fields:
+  ** *Base URL*
+  ** *Name*
+  ** *Value*
+  ** *Sent As*: Select one of the following values:
+    *** *Header*: If selected, Composer adds the following header to the request: `Name: Value`.
+    *** *Query Parameter*: If selected, Composer adds the following query parameter to the request: `Name=Value`.
+
+* *Basic Authentication* +
+  The Basic Authentication method enables an HTTP user agent to provide a username and password when making an HTTP request. Provide a value for the following fields:
+  ** *Base URL*
+  ** *Username*
+  ** *Password*
+
+* *Bearer Token Authorization*
+  Bearer Token Authorization is the authentication process that sends a header named Authorization with a value of Bearer and a token with every request. Provide a value for the following fields:
+  ** *Base URL*
+  ** *Token*
+
+=== Test the Connection
+
+After a connection method is established, you can then test the connection. While other connectors for Composer are tested automatically in the background, the HTTP Connector is tested manually. You can decide which page to use for the test. Testing the connection is important because you cannot edit the connection later.
+
+To test the connection, provide a value for the following fields:
+
+* *Request Method*
+* *Request Path*: The value in this field is relative to the Base URL.
+* *Request Headers*
+  ** *Key*
+  ** *Value*
+* *Request Body*: This option appears when the HTTP method is POST and is the body of the request message. 
+
+Click *Connect* to test the connection. A message appears to indicate the connection was created successfully (which is indicated by a response code of `200`) or that an error occurred.
+
+Note that if the application requires a token exchange to transact data, the user will need to create a token for each use case. For example, in Salesforce Marketing Cloud, this is needed because the first step in Flow 1 will generate a token, which might get invalidated by a similar step in Flow 2.
```

Comment:
```suggestion
Note that if the application requires a token exchange to transact data, you need to create a token for each use case. For example, in Salesforce Marketing Cloud, this is needed because the first step in Flow 1 will generate a token, which might get invalidated by a similar step in Flow 2.
```